translations/es/pack/mut_gen.json

```diff
@@ -311,7 +311,7 @@
 		"name": "Magneto",
 		"subname": "Erik Lensherr",
 		"text": "Victory 1.",
-		"traits": "Brotherhood of Mutantes.",
```

Comment:
Just a thought here, since this is in the translations, `Mutantes` is probably correct.

Reply:
Thanks, I changed it back for Magneto in this file but I then found a `Mutantee.` that I updated to `Mutante.` in this file.